TradeItIosTicketSDK2/TradeItLinkedBrokerAccount.swift

```diff
@@ -171,19 +171,18 @@
     
     public func cancelOrder(orderNumber: String,
                             onSuccess: @escaping () -> Void,
-                            onSecurityQuestion: @escaping (
-                                TradeItSecurityQuestionResult,
-                                _ submitAnswer: @escaping (String) -> Void,
-                                _ onCancelSecurityQuestion: @escaping () -> Void
+                            onVerifyUrl: @escaping (
+                                                URL,
+                                                _ complete1FA: @escaping () -> Void
```

Comment:
Indentation looks incorrect here. Also why was the `onSecurityQuestion` removed? Was that only used by CIMB?

Reply:
Yes security question was added specifically for CIMB